controllers/controller.games.js

```diff
@@ -42,4 +42,18 @@ exports.getComments = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+};
+
+exports.postComments = (req, res, next) => {
+    const ID = req.params.review_id;
+    const promises = [insertComments(req.body, ID),selectReviewID(ID)]
+
+    Promise.all(promises)
+    .then((comments) => {
+        comments = comments[0]
+        res.status(201).send({comments})
```

Comment:
Good choice of naming, It's small but I'd slightly change this to comment as were only sending back a single comment